roles/postgres_upgrade/tasks/configure.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+- name: disable old database auto-start
+  lineinfile:
+    dest: "/etc/postgresql/{{ pg_old }}/main/start.conf"
+    regexp: "^auto"
+    line: "manual"
+  become: yes
+
+- name: set new cluster port to 5432
+  lineinfile:
+    dest: "/etc/postgresql/{{ pg_new }}/main/postgresql.conf"
+    regexp: "^port"
+    line: "port = 5432"
+  become: yes
+
+- name: copy pg_hba.conf
+  copy:
+    src: "/etc/postgresql/{{ pg_old }}/main/pg_hba.conf"
+    dest: "/etc/postgresql/{{ pg_new }}/main/pg_hba.conf"
+    remote_src: yes
+    force: yes
+  become: yes
+
+- name: restart postgres
+  service:
+    name: postgresql
+    state: restarted
+  become: yes
+
+- name: rebuild stats and indexes # noqa 301
```

Comment:
AFAIK this command will only vacuum but not reindex. That would require https://www.postgresql.org/docs/9.3/sql-reindex.html

Reply:
Yeah maybe the wording is a bit inaccurate :+1:

It rebuilds the internal statistics used for query optimisation (which are wiped out in the upgrade) in several passes: https://www.postgresql.org/docs/10/app-vacuumdb.html

Basically after `pg_upgrade` finishes, it spits out a bash script called `analyze_new_cluster.sh` and recommends you run it, and the script just contains some notes and this line:
```
/usr/lib/postgresql/10/bin/vacuumdb --all --analyze-in-stages
```

So I thought this would be simpler and more explicit that calling he script directly.